dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
       },
       {
         "value": "626GH",
-        "label": "COOKEVILLE VA CLINIC"
+        "label": "Cookeville VA Clinic"
       },
       {
         "value": "614GF",
-        "label": "NONCONNAH BOULEVARD VA CLINIC"
+        "label": "Nonconnah Boulevard VA Clinic"
       },
       {
         "value": "614GE",
-        "label": "COVINGTON VA CLINIC"
+        "label": "Covington VA Clinic"
       },
       {
         "value": "621BY",
-        "label": "WILLIAM C. TALLENT VA OPC"
+        "label": "William C. Tallent Department of Veterans Affairs Outpatient Clinic"
       },
       {
         "value": "614GG",
-        "label": "JACKSON VA CLINIC"
+        "label": "Jackson VA Clinic"
       },
       {
         "value": "614GH",
         "label": "BOLIVAR CBOC"
       },
       {
         "value": "614GI",
-        "label": "DYERSBURG VA CLINIC"
+        "label": "Dyersburg VA Clinic"
       },
       {
         "value": "621GI",
-        "label": "DANNIE A. CARR VETERANS OPC"
+        "label": "Dannie A. Carr Veterans Outpatient Clinic"
       },
       {
         "value": "626GM",
-        "label": "MAURY COUNTY CBOC"
+        "label": "Maury County VA Clinic"
       },
       {
         "value": "626GL",
-        "label": "ROANE COUNTY VA CLINIC"
+        "label": "Roane County VA Clinic"
       },
       {
         "value": "626GK",
-        "label": "MCMINNVILLE VA CLINIC"
+        "label": "McMinnville VA Clinic"
       },
       {
         "value": "621GG",
-        "label": "MORRISTOWN VA CLINIC"
+        "label": "Morristown VA Clinic"
       },
       {
         "value": "621GK",
-        "label": "CAMPBELL COUNTY VA CLINIC"
+        "label": "Campbell County VA Clinic"
       },
       {
         "value": "626QA",
-        "label": "ALBION STREET VA CLINIC"
+        "label": "Albion Street VA Clinic"
       },
       {
         "value": "626QB",
-        "label": "CHARLOTTE AVENUE VA CLINIC"
+        "label": "Charlotte Avenue VA Clinic"
       },
       {
         "value": "626QC",
-        "label": "POINTE CENTRE VA CLINIC"
+        "label": "Pointe Centre VA Clinic"
       },
       {
         "value": "621QE",
-        "label": "DOWNTOWN WEST VA CLINIC"
+        "label": "Downtown West VA Clinic"
       },
       {
         "value": "626GP",
-        "label": "GALLATIN VA CLINIC"
+        "label": "Gallatin VA Clinic"
       },
       {
         "value": "626GO",
-        "label": "INTERNATIONAL PLAZA VA CLINIC"
+        "label": "International Plaza VA Clinic"
       },
       {
         "value": "614",
-        "label": "MEMPHIS VA MEDICAL CENTER"
+        "label": "Memphis VA Medical Center"
       },
       {
         "value": "621",
-        "label": "JAMES H. QUILLEN VAMC"
+        "label": "James H. Quillen Department of Veterans Affairs Medical Center"
       },
       {
         "value": "626",
-        "label": "TENNESSEE VALLEY HCS"
+        "label": "Nashville VA Medical Center"
```

Comment:
Let's confirm this name change.

Reply:
This is the correct facility since "Nashville VA Medical Center" on [Facility Locator](https://www.va.gov/find-locations/facility/vha_626) points to [TENNESSEE VALLEY HCS](https://www.tennesseevalley.va.gov/locations/directions.asp).

Reply:
Flagging this label since it goes from an entire health care system to a single facility.